### PR TITLE
Adding support for basic logic operators

### PR DESCRIPTION
**Pull request by narendasan**
# Description

This adds support for the following logic operators (h/t @inocsin)

- aten::gt
- aten::lt
- aten::le
- aten::ge
- aten::eq

## Type of change

Please delete options that are not relevant and/or add your own.

- New feature (non-breaking change which adds functionality)

# Checklist:

- [X] My code follows the style guidelines of this project (You can use the linters)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas and hacks
- [X] I have made corresponding changes to the documentation
- [X] I have added tests to verify my fix or my feature
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: core/conversion/converters/impl/element_wise.cpp [hunk 1]
```diff
@@ -352,6 +352,191 @@ auto element_wise_registrations TRTORCH_UNUSED =
...
+                    TRTORCH_CHECK(gt, "Unable to create greater layer from node: " << *n);
+
+                    gt->setName(util::node_info(n).c_str());
+                    auto out = ctx->AssociateValueAndTensor(n->outputs()[0], gt->getOutput(0));
+                    LOG_DEBUG("Output tensor shape: " << out->getDimensions());
+                    return true;
+                  }})
+        .pattern({"aten::gt.Scalar(Tensor self, Scalar other) -> (Tensor)",
+                  [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+                    // TODO: Remove with functionalization
```
Comment (peri044) [inline core/conversion/converters/impl/element_wise.cpp:374]: @inocsin What does this TODO comment mean ?

**Review by peri044 — APPROVED** [review]

File: core/conversion/converters/impl/element_wise.cpp (hunk 1 above)
Comment (inocsin) [reply core/conversion/converters/impl/element_wise.cpp:374]: In fact, i'm not sure either. I see every converter in element_wise.cpp has the same comment, maybe the first contributer to element_wise.cpp knows this?

Comment (narendasan) [reply core/conversion/converters/impl/element_wise.cpp:374]: all of the _ operators like `aten::add_` are inplace operators they should not be used after we added functionalization but we havent removed them yet

Comment (peri044) [reply core/conversion/converters/impl/element_wise.cpp:374]: Got it

**Pull request merged**